pkg/cmd/generate.go

```
@@ -76,3 +79,27 @@ func Generate(paths []string) error {
 	}
 	return nil
 }
+
+func validateOutput(data []byte) {
```

yep, let's return an error here and capture it wherever it is being called